web/templates/tooltip_templates.hbs

```diff
@@ -18,7 +18,7 @@
     {{t 'Reply to selected conversation' }}
     {{tooltip_hotkey_hints "R"}}
 </template>
-<template id="compose_reply_button_disabled_tooltip_template">
+<template id="compose_reply_direct_disabled_button_tooltip_template">
     {{t 'You are not allowed to send direct messages in this organization.' }}
```

Comment:
You can't rename this without also changing the reference in `web/src/compose_closed_ui.js` in the same commit.

Otherwise, this commit is broken alone, which makes it hard to verify your work -- one really want to be able to check that an individual commit is fully correct before moving on to reading the next one. 
Check out the [Zulip commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details.

Reply:
I'm sorry I missed that, I've made the change.